sashimi-webapp/src/logic/inputHandler/DocumentPrinter/index.js

```diff
@@ -0,0 +1,61 @@
+import elementUtils from 'src/helpers/elementUtils';
+import core from './core';
+
+const keyEventListener = function keyEventListener(event) {
+  if (event.key === 'p' && (event.ctrlKey || event.metaKey)) {
+    // metaKey will match Window key and Mac's CMD key
```

Comment:
It was a real issue a few years back before 2012.

However, on modern browser MetaKey is mostly assigned to CMD in Mac.
See https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Also, check the more recent answer in the stackoverflow link you provided http://stackoverflow.com/questions/3902635/how-does-one-capture-a-macs-command-key-via-javascript?answertab=active#comment43996640_3922353